src/PowerShellEditorServices/Services/Workspace/LanguageServerSettings.cs

```diff
@@ -22,6 +22,7 @@ internal class LanguageServerSettings
         public CodeFoldingSettings CodeFolding { get; set; }
         public PesterSettings Pester { get; set; }
         public string Cwd { get; set; }
+        public bool EnableReferencesCodeLens { get; set; }
```

Comment:
LMAO this defaulted to `false` which caused the unit test `CanSendReferencesCodeLensRequestAsync` to fail. Should we just default to `true` for now?